src/cascade/testing_utilities/compare_dismod_db.py

```diff
@@ -0,0 +1,45 @@
+from difflib import SequenceMatcher
+from sqlite3 import connect
+
+
+SQLITE_TABLES = "SELECT name FROM sqlite_master WHERE type='table';"
+
+
+class CompareDatabases:
+    def __init__(self, db_file_a, db_file_b):
+        self.conn = [connect(str(dbf)) for dbf in (db_file_a, db_file_b)]
+        self.tables = [sorted([x[0] for x in connl.execute(SQLITE_TABLES)]) for connl in self.conn]
+
+    def table_diffs(self):
+        sm = SequenceMatcher()
+        sm.set_seqs(*self.tables)
+        return [op for op in sm.get_opcodes() if op[0] != "equal"]
+
+    def different_tables(self):
+        """Which tables are different, except for the log, which always differs."""
+        common = sorted(set(self.tables[0]) & set(self.tables[1]))
```

Comment:
Are tables which are absent in one database not considered different?

Reply:
The `table_diffs` tells you which tables are added or deleted. This one tells you which of existing tables have differences that you can probe into. It makes more sense in the way it's used.